fontmaker/core/ttfparser.go

```diff
@@ -74,7 +74,7 @@ type TTFParser struct {
 	groupingTables []CmapFormat12GroupingTable
 
 	//data of font
-	cacheFontData []byte
+	CacheFontData []byte
```

Comment:
```suggestion
	CachedFontData []byte
```
Otherwise it sounds like a command haha